lib/vanagon/platform/windows.rb

```diff
@@ -163,9 +163,24 @@ def generate_nuget_package(project)
       # @return [Array] list of commands required to build an msi package for the given project from a tarball
       def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
-        cg_name = "ProductComponentGroup"
-        wix_extensions =  "-ext WiXUtilExtension -ext WixUIExtension"
-        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} #{wix_extensions}"
+        wix_extensions = "-ext WiXUtilExtension -ext WixUIExtension"
+        # Heat command documentation at: http://wixtoolset.org/documentation/manual/v3/overview/heat.html
+        #   dir <directory> - Traverse directory to find all sub-files and directories.
+        #   -ke             - Keep Empty directories
+        #   -cg             - Component Group Name
+        #   -gg             - Generate GUIDS now
+        #   -srd            - Suppress root element generation, we want to reference one of the default root elements
+        #                     INSTALLDIR or APPDATADIR in the directorylist.wxs file, not a newly generated one.
+        #   -sreg           - Suppress registry harvesting.
+        #   -dr             - Root DirectoryRef to point all components to
+        #   -var            - Replace "SourceDir" in the @source attributes of all components with a preprocessor variable
+        app_heat_flags = " -dr INSTALLDIR -v -ke -indent 2 -cg AppComponentGroup -gg -srd -t wix/filter.xslt -sreg -var var.AppSourcePath "
+        app_source_path = "SourceDir/#{project.settings[:base_dir]}/#{project.settings[:company_id]}/#{project.settings[:product_id]}"
```

Comment:
Part of me wants to put checks in to make sure the user has at least these three variables set, so the build fails early. That's not a requirement for this PR though